setup/webmail.sh

```diff
@@ -28,10 +28,20 @@ apt_install \
 # Install Roundcube from source if it is not already present or if it is out of date.
 # Combine the Roundcube version number with the commit hash of plugins to track
 # whether we have the latest version of everything.
+
+# Plugin versions:
+# Latest version of persistent_login is 5.2.0
+# Latest version of HTML5_Notifier is 0.6.4
+#      Maintainer has migrated GitHub Profiles
+# Latest version of CardDAV is 4.0.4
+#      A new maintainer is developing the plugin
+#      4.0.4 is in a different compression format (.tgz)
```

Comment:
Should list only the version being installed and not future versions.  There was work when carddav was updated before and will likely be needed again (i.e. a standalone PR)